scripts/lib/libvirt/admin-config

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+import argparse
+
+import libvirt_setup
+
+
+def main():
+
+    parser = argparse.ArgumentParser(description="Create Admin Node Config.")
+    parser.add_argument("cloud", type=str, help="Name of the cloud")
+    parser.add_argument("adminnodememory", type=str, help="Amount of Memory")
+    parser.add_argument("adminvcpus", type=str, help="Number of VCPUs")
+    parser.add_argument("emulator", type=str, help="Emulator")
```

Comment:
this is usually ` /usr/bin/qemu`, `/usr/bin/qemu-system-x86_64` and so on... if you look for `Emulator` in the libvirt context you will find it quickly, that's why I didn't provide an Example, but anyways I can try to enhance the help output a bit overall